packages/common/src/wallet-adapters/solong/index.tsx

```diff
@@ -1,70 +0,0 @@
-import EventEmitter from 'eventemitter3';
-import { PublicKey, Transaction } from '@solana/web3.js';
-import { WalletAdapter } from '@solana/wallet-base';
-import { notify } from '../../utils/notifications';
-
-export class SolongWalletAdapter extends EventEmitter implements WalletAdapter {
-  _publicKey: PublicKey | null;
-  _onProcess: boolean;
-  constructor() {
-    super();
-    this._publicKey = null;
-    this._onProcess = false;
-    this.connect = this.connect.bind(this);
-  }
-
-  get publicKey() {
-    return this._publicKey;
-  }
-
-  async signTransaction(transaction: Transaction) {
-    return (window as any).solong.signTransaction(transaction);
-  }
-
-  async signAllTransactions(transactions: Transaction[]) {
-    const solong = (window as any).solong;
-
-    // Temp. workaround to ensure requests to sign multiple transactions at once don't fail when Solong wallet is used
-    // Signing transactions one by one as a fallback works but the UX is not great because user is asked to sign multiple times
-    for (let t of transactions) {
```

Comment:
This was not the best UX but it did work as a fallback. After the change trying to sign multiple transactions with solong throws error. 
You can reproduce by starting governance [devtools](http://localhost:3000/#/devtools) and trying to generate artefacts with Solong wallet

Reply:
Great, thanks for finding this. I'll sort it out.

Reply:
This should be resolved by this change: https://github.com/solana-labs/wallet-adapter/blame/master/packages/wallets/solong/src/adapter.ts#L108 in https://github.com/solana-labs/wallet-adapter/commit/183988deca18478c2d1408a3106288c00cfe5deb

The issue is that the function was expecting Solong to return the transaction object, and it returns undefined. This is fixed by all wallets returning the signed transaction object if the wallet returns nothing.